test/AWSMetadataUtilities.jl

```diff
@@ -0,0 +1,448 @@
+function _clean_high_level_definition(definition::String)
+    # Required post Julia1.5 with how triple quoted strings are dealt with.
+    definition = replace(definition, " "=>"")
+    definition = replace(definition, "\n"=>"")
```

Comment:
Is the comment referring to Julia 1.5 and after or release after Julia 1.5?
```suggestion
    # Required post Julia 1.5 with how triple quoted strings are dealt with.
    definition = replace(definition, " " => "")
    definition = replace(definition, "\n" => "")
```